sdk/confidentialledger/azure-security-confidentialledger/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
```

Comment:
please check if you use live test or recorded test.

recorded test does not need this and test-resources.json

Reply:
I am adding these now so that we can use the pipeline in the future. Is that ok, or should I hold off?